model/model.go

```diff
@@ -57,7 +57,8 @@ func (model Model) AddDef(sec string, key string, value string) bool {
 	ast.Value = value
 
 	if sec == "r" || sec == "p" {
-		ast.Tokens = strings.Split(ast.Value, ", ")
+		newValue := strings.Replace(ast.Value, " ", "", -1)
+		ast.Tokens = strings.Split(newValue, ",")
 		for i := range ast.Tokens {
 			ast.Tokens[i] = key + "_" + ast.Tokens[i]
```

Comment:
Better to use `strings.Replace` here. 
WDYT?

Reply:
I think it will be more clear if we use 
```go
	act.Tokens = strings.FieldsFunc(ast.Value, func(c rune) bool {
		return !unicode.IsLetter(c) && !unicode.IsNumber(c)
	})
```
So that users can use whatever sep string they like and that won't affect the further logic.

Reply:
@jonyhy96, thanks for your suggestion, but, I think, we shouldn't allow users to change the actual syntax and semantic any further, I think this much liberty is way enough for users.

Reply:
@jonyhy96 The commits need rebase for keep clean.

Reply:
Sorry @jonyhy96 , I should be at @DivyPatel9881 .